components/LessonsChallenges.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react'
+import '../scss/lessonsChallenges.scss'
+
+type ChallengeStatusProps = {
+  challengesData: Challenge[]
+}
+
+type LessonChallengeProps = {
+  lessons: LessonChallenge[]
+}
+
+type LessonChallenge = {
+  order: number
+  title: string
+  challenges: Challenge[]
+}
+
+type Challenge = {
+  challengeNumber: number
+  challengeStatus?: string
+}
+
+export const ChallengeStatus: React.FC<ChallengeStatusProps> = props => {
+  const displayChallengeStatus = props.challengesData.map(
+    (eachChallenge, challengeId) => {
+      let challengeStatus = 'default_challenge_status'
+      if (eachChallenge.challengeStatus === 'passed') {
+        challengeStatus = 'passed_challenge_status'
+      }
+      if (eachChallenge.challengeStatus === 'wrong') {
+        challengeStatus = 'unapproved_challenge_status'
```

Comment:
keep it consistent, `fail_challenge_status`

`unapproved` could cause confusion because `unapproved` means 2 things:
1. Haven't been reviewed yet.
2. fail

Reply:
`wrong_challenge_status` or `fail_challenge_status`. If you change it to `fail`, please update your `if` check to do: `eachChallenge.challengeStatus === 'fail'` so that it is consistent.